data/json/items/gun/50.json

```diff
@@ -70,5 +70,29 @@
       "barrel_length": -1
     },
     "flags": [ "RELOAD_EJECT" ]
+  },
+  {
+    "id": "spr_2",
+    "copy-from": "rifle_base",
+    "type": "GUN",
+    "name": "Pindad SPR 2",
+    "name_plural": "Pindad SPR-2",
+    "description": "Bolt-action, Anti Material-rifle designed by Pindad Armaments, its medium size and weight is offset by its range and damage.",
```

Comment:
```suggestion
    "description": "Bolt-action anti-materiel rifle designed by Pindad Armaments, its medium size and weight is offset by its range and damage.",
```

Reply:
thanks, blame it to my auto-capital keyboard function, it is a typo-maker.